QueryBuilder/Compilers/Compiler.cs

```diff
@@ -777,8 +777,18 @@ public virtual string WrapValue(string value)
             return opening + value.Replace(closing, closing + closing) + closing;
         }
 
-        public virtual string Parameter<T>(SqlResult ctx, T value)
+        public virtual string Parameter<T>(SqlResult ctx, T value) 
         {
+            if (value is AbstractFunctionValue)
+            {
+                if (value is NullifValue)
```

Comment:
change to ->  if (value is NullifValue **nullifValue**)

Reply:
C# 7.1 support is not enabled in the project.